starter-core/src/main/java/io/micronaut/starter/feature/github/workflows/docker/templates/mavenJavaWorkflow.rocker.raw

```diff
@@ -0,0 +1,34 @@
+@import io.micronaut.starter.options.JdkVersion
+
+@args (
+JdkVersion jdkVersion
+)
+
+name: Java CI
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@@v2
+      - uses: actions/cache@@v2
+        with:
+          path: |
+            ~/.m2
+          key: ${{ runner.os }}-maven-test-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-test-
+      - name: Set up JDK @jdkVersion.majorVersion()
+        uses: actions/setup-java@@v1
+        with:
+          java-version: @jdkVersion.majorVersion()
+      - name: Docker login
+        uses: docker/login-action@@v1
+        with:
+          registry: ${{ secrets.DOCKER_REGISTRY_URL }}
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Build and push docker image
+        env:
+          DOCKER_REGISTRY: ${{ secrets.DOCKER_REGISTRY }}
+        run: ./mvnw clean package jib:build
```

Comment:
No need to clean, or @aalmiray will appear here 😅

Reply:
oh sorry for this one, copy-paster left over